src/components/Collection.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 
-const Collection = ({active, collection, index, handleDelete, handleActive}) => {
+const Collection = ({active, name, index, handleDelete, handleActive}) => {
   return (
     <div
       className={'h-12 flex flex-row items-center hover:cursor-pointer' + (index === 0 ? '' : ' border-t border-gray-400')}
-      onClick={() => handleActive(collection.name)}
+      onClick={handleActive}
     >
       <div className='flex-none flex flex-row items-center py-4 px-4 w-1/2'>
         <p className='flex-none text-gray-600 text-md'>Name:</p>
-        <p className={'flex-auto font-semibold text-md ml-1 ' + (active ? 'text-redorange' : 'text-bdazzledblue')}>{collection.name}</p>
+        <p className={'flex-auto font-semibold text-md ml-1 ' + (active ? 'text-redorange' : 'text-bdazzledblue')}>{name}</p>
       </div>
-      <div className='flex-auto flex flex-col items-end py-4 pr-4 stroke-redorange' onClick={() => handleDelete(collection.name)}>
-        <svg xmlns="http://www.w3.org/2000/svg" className='hover:cursor-pointer' width="20" viewBox="0 0 24 24" fill="none" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
+      <div className='flex-auto flex flex-col items-end py-4 pr-4 stroke-redorange'>
+        <svg onClick={handleDelete} className='hover:cursor-pointer' xmlns="http://www.w3.org/2000/svg" width="20" viewBox="0 0 24 24" fill="none" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
```

Comment:
moved onClick into svg.  The div takes up almost half of the space, so this seems to improve UX significantly